tests/ActivityApiTest.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+use Rogue\Models\Signup;
+use Rogue\Models\Event;
+use Faker\Generator;
+
+class ActivityApiTest extends TestCase
+{
+    /*
+     * Base URL for the Api.
+     */
+    protected $activityApiUrl = 'api/v2/activity';
+
+    /**
+     * Test for retrieving a user's activity with limit query param.
+     *
+     * GET /activity?limit=8
+     * @return void
+     */
+    public function testActivityIndexWithLimitQuery()
+    {
+        $response = $this->json('GET', $this->activityApiUrl . '?limit=8');
```

Comment:
The `$this->json` method returns the `TestCase` as a "self" reference, so it shouldn't be saved as `$response`.